exchanges/lbank/lbank.go

```diff
@@ -63,12 +64,12 @@ const (
 
 // GetTicker returns a ticker for the specified symbol
 // symbol: eth_btc
-func (l *Lbank) GetTicker(ctx context.Context, symbol string) (TickerResponse, error) {
+func (l *Lbank) GetTicker(ctx context.Context, symbol string) (*TickerResponse, error) {
 	var t TickerResponse
 	params := url.Values{}
 	params.Set("symbol", symbol)
 	path := fmt.Sprintf("/v%s/%s?%s", lbankAPIVersion1, lbankTicker, params.Encode())
```

Comment:
Suggestion/OOS: Concat all these calls instead, maybe defer to Jules 😆

Reply:
Yeah I started doing this but then noticed all the oudated endpoints too, error encapsulation potential improvements, then figured I'd just focus on the test issue instead (the original task) 😅